### ISSUE TITLE
Scale breaks for some charts in HTML report

### ISSUE THREAD

**Issue by claudiobrandt** [allinurl/goaccess#1178]
Hi, 

On issue #933, @anarcat suggested using logarithm scales as a way to make some charts more visually meaningful. I'd like to suggest that you consider scale breaks instead of (or in addition to) logarithm scales. 

This blog post describes the issue being faced here, that of graphing highly skewed data, and proposes some alternatives, with both the logarithm scale and the scale break among them: [https://tomhopper.me/2010/08/30/graphing-highly-skewed-data/](url)

Both approaches have pros and cons. My opinion is that logarithm scales are hard to interpret for people without strong analytical/math skills. In any case, if scale breaks are hard to implement, or if logarithm scale is thought of as the best of the two approaches, then log scales would certainly be much better the the as-is alternative:

![image](https://user-images.githubusercontent.com/30415322/42967777-b2c6a67a-8b77-11e8-8e38-dfb348ab388f.png)

Thank you for writing this beautiful software!
Claudio

**Comment by allinurl** [comment]
I agree that scale break might be easier to read for some folks. I can look into this. Thanks for the suggestion!
